pkg/apis/servicecatalog/types.go

```diff
@@ -226,8 +226,41 @@ type ServiceClass struct {
 	AlphaRequires []string
 }
 
+type Plan struct {
+	// PlanRef is a reference to the full plan details.
+	PlanRef v1.LocalObjectReference `json:"planRef"`
```

Comment:
I'm not sure what Plan is vs. ServicePlan - I had thought we agreed to just split them rather than keeping some information in two places.